main.c

```diff
@@ -7,16 +7,78 @@
 #include "pak_get_file_type.h"
 #include "pak_header.h"
 #include "pak_pack.h"
+#include <stdio.h>

```

Comment:
`stdio.h` is already included in [pak_file_io.h](https://github.com/myfreeer/chrome-pak-customizer/blob/bfabc033207ebbd6e0017ce99500d3e379a0a3f6/pak_file_io.h#L7) by [pak_pack.h](https://github.com/myfreeer/chrome-pak-customizer/blob/bfabc033207ebbd6e0017ce99500d3e379a0a3f6/pak_pack.h#L18).